.github/workflows/ci-linux.yml

```diff
@@ -0,0 +1,139 @@
+# **********************************************************
+# Copyright (c) 2020 Google, Inc.  All rights reserved.
+# **********************************************************
+
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice,
+#   this list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of Google, Inc. nor the names of its contributors may be
+#   used to endorse or promote products derived from this software without
+#   specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL VMWARE, INC. OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+# DAMAGE.
+
+
+# Github Actions workflow for Continuous Integration testing on Linux.
+# TODO i#4549: This is a work-in-progress and not enabled currently.
+
+name: ci-linux
+on:
+  # Run on pushes to any repository branch.
+  # TODO i#4549: Uncomment when Github Actions integration complete.
+  # push:
+
+  # Manual trigger using the Actions page. May remove when integration complete.
+  workflow_dispatch:
+
+jobs:
+  x86:
+    runs-on: ubuntu-16.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Create Build Environment
+      run:
+        sudo apt-get -y install doxygen vera++ cmake zlib1g-dev libsnappy-dev;
+        sudo apt-get -y install g++-multilib;
+
+    - name: Run Suite
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: ./suite/runsuite_wrapper.pl travis $EXTRA_ARGS
+      env:
+        DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: no
+        DEPLOY: no
```

Comment:
This is used only within .travis.yml to talk to the deployment step.  Is an env var also the way to communicate between steps here?  If not sure, put a comment here so we can remove this if it's not the way to go when we add deployment.

Ditto below.

Reply:
I haven't looked at that yet. As we haven't ported the deploy step at all yet, I've just removed it for now.